



<!DOCTYPE html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      <meta http-equiv="x-ua-compatible" content="ie=edge">
      
        <meta name="description" content="Transcriptional Plasticity in the Hippocampus and its Role in Conditioned Avoidance Learning">
      
      
        <link rel="canonical" href="http://raynamharris.github.io/Thesis/01_intwt15/">
      
      
        <meta name="author" content="Rayna Harris">
      
      
        <meta name="lang:clipboard.copy" content="Copy to clipboard">
      
        <meta name="lang:clipboard.copied" content="Copied to clipboard">
      
        <meta name="lang:search.language" content="en">
      
        <meta name="lang:search.pipeline.stopwords" content="True">
      
        <meta name="lang:search.pipeline.trimmer" content="True">
      
        <meta name="lang:search.result.none" content="No matching pages">
      
        <meta name="lang:search.result.one" content="1 matching page">
      
        <meta name="lang:search.result.other" content="# matching page">
      
        <meta name="lang:search.tokenizer" content="[\s\-]+">
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.0, mkdocs-material-3.0.3">
    
    
      
        <title>Chapter 1 Brains and behavior - Rayna Harris Ph.D. Thesis</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/application.451f80e5.css">
      
        <link rel="stylesheet" href="../assets/stylesheets/application-palette.22915126.css">
      
      
        
        
        <meta name="theme-color" content="#009688">
      
    
    
      <script src="../assets/javascripts/modernizr.1aa3b519.js"></script>
    
    
      <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
      
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700|Roboto+Mono">
        <style>body,input{font-family:"Roboto","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
      
    
    <link rel="stylesheet" href="../assets/fonts/material-icons.css">
    
    
  </head>
  
    
    
    <body dir="ltr" data-md-color-primary="teal" data-md-color-accent="teal">
  
    <svg class="md-svg">
      <defs>
        
        
          <svg xmlns="http://www.w3.org/2000/svg" width="416" height="448"
    viewBox="0 0 416 448" id="__github">
  <path fill="currentColor" d="M160 304q0 10-3.125 20.5t-10.75 19-18.125
        8.5-18.125-8.5-10.75-19-3.125-20.5 3.125-20.5 10.75-19 18.125-8.5
        18.125 8.5 10.75 19 3.125 20.5zM320 304q0 10-3.125 20.5t-10.75
        19-18.125 8.5-18.125-8.5-10.75-19-3.125-20.5 3.125-20.5 10.75-19
        18.125-8.5 18.125 8.5 10.75 19 3.125 20.5zM360
        304q0-30-17.25-51t-46.75-21q-10.25 0-48.75 5.25-17.75 2.75-39.25
        2.75t-39.25-2.75q-38-5.25-48.75-5.25-29.5 0-46.75 21t-17.25 51q0 22 8
        38.375t20.25 25.75 30.5 15 35 7.375 37.25 1.75h42q20.5 0
        37.25-1.75t35-7.375 30.5-15 20.25-25.75 8-38.375zM416 260q0 51.75-15.25
        82.75-9.5 19.25-26.375 33.25t-35.25 21.5-42.5 11.875-42.875 5.5-41.75
        1.125q-19.5 0-35.5-0.75t-36.875-3.125-38.125-7.5-34.25-12.875-30.25-20.25-21.5-28.75q-15.5-30.75-15.5-82.75
        0-59.25 34-99-6.75-20.5-6.75-42.5 0-29 12.75-54.5 27 0 47.5 9.875t47.25
        30.875q36.75-8.75 77.25-8.75 37 0 70 8 26.25-20.5
        46.75-30.25t47.25-9.75q12.75 25.5 12.75 54.5 0 21.75-6.75 42 34 40 34
        99.5z" />
</svg>
        
      </defs>
    </svg>
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" data-md-component="overlay" for="__drawer"></label>
    
      <a href="../#chapter-1-avoidance-learning-induces-strong-up-regulation-of-transcription-in-the-dg-subfield-of-the-dorsal-hippocampus" tabindex="1" class="md-skip">
        Skip to content
      </a>
    
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid">
    <div class="md-flex">
      <div class="md-flex__cell md-flex__cell--shrink">
        <a href="http://raynamharris.github.io/Thesis" title="Rayna Harris Ph.D. Thesis" class="md-header-nav__button md-logo">
          
            <i class="md-icon"></i>
          
        </a>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        <label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label>
      </div>
      <div class="md-flex__cell md-flex__cell--stretch">
        <div class="md-flex__ellipsis md-header-nav__title" data-md-component="title">
          
            
              <span class="md-header-nav__topic">
                Rayna Harris Ph.D. Thesis
              </span>
              <span class="md-header-nav__topic">
                Chapter 1 Brains and behavior
              </span>
            
          
        </div>
      </div>
      <div class="md-flex__cell md-flex__cell--shrink">
        
          
            <label class="md-icon md-icon--search md-header-nav__button" for="__search"></label>
            
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" placeholder="Search this site" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active">
      <label class="md-icon md-search__icon" for="__search"></label>
      <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">
        &#xE5CD;
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="result">
          <div class="md-search-result__meta">
            Type to start searching this site
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
          
        
      </div>
      
        <div class="md-flex__cell md-flex__cell--shrink">
          <div class="md-header-nav__source">
            


  


  <a href="https://github.com/raynamharris/Thesis" title="Go to repository" class="md-source" data-md-source="github">
    
    <div class="md-source__repository">
      raynamharris/Thesis
    </div>
  </a>

          </div>
        </div>
      
    </div>
  </nav>
</header>
    
    <div class="md-container">
      
        
      
      <main class="md-main">
        <div class="md-main__inner md-grid" data-md-component="container">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" data-md-level="0">
  <label class="md-nav__title md-nav__title--site" for="__drawer">
    <a href="http://raynamharris.github.io/Thesis" title="Rayna Harris Ph.D. Thesis" class="md-nav__button md-logo">
      
        <i class="md-icon"></i>
      
    </a>
    Rayna Harris Ph.D. Thesis
  </label>
  
    <div class="md-nav__source">
      


  


  <a href="https://github.com/raynamharris/Thesis" title="Go to repository" class="md-source" data-md-source="github">
    
    <div class="md-source__repository">
      raynamharris/Thesis
    </div>
  </a>

    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="Home" class="md-nav__link">
      Home
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../preface/" title="Preface" class="md-nav__link">
      Preface
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../00_intro/" title="Introduction" class="md-nav__link">
      Introduction
    </a>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-toggle md-nav__toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        Chapter 1 Brains and behavior
      </label>
    
    <a href="./" title="Chapter 1 Brains and behavior" class="md-nav__link md-nav__link--active">
      Chapter 1 Brains and behavior
    </a>
    
      
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#chapter-1-avoidance-learning-induces-strong-up-regulation-of-transcription-in-the-dg-subfield-of-the-dorsal-hippocampus" title="Chapter 1:  Avoidance learning induces strong up-regulation of transcription in the DG subfield of the dorsal hippocampus" class="md-nav__link">
    Chapter 1:  Avoidance learning induces strong up-regulation of transcription in the DG subfield of the dorsal hippocampus
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#abstract" title="Abstract" class="md-nav__link">
    Abstract
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#introduction" title="Introduction" class="md-nav__link">
    Introduction
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#materials-and-methods" title="Materials and Methods" class="md-nav__link">
    Materials and Methods
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#the-active-place-avoidance-task" title="The Active Place Avoidance Task" class="md-nav__link">
    The Active Place Avoidance Task
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#statistical-analyses-of-behavior" title="Statistical analyses of behavior" class="md-nav__link">
    Statistical analyses of behavior
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-11-experimental-design" title="Fig. 1.1: Experimental design." class="md-nav__link">
    Fig. 1.1: Experimental design.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tissue-preparation-from-dg-ca3-and-ca1-subfields" title="Tissue preparation from DG, CA3, and CA1 subfields" class="md-nav__link">
    Tissue preparation from DG, CA3, and CA1 subfields
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-sequencing-and-bioinformatics" title="RNA-sequencing and bioinformatics" class="md-nav__link">
    RNA-sequencing and bioinformatics
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#statistics-and-data-visualization-of-rna-sequencing-data" title="Statistics and data visualization of RNA-sequencing data" class="md-nav__link">
    Statistics and data visualization of RNA-sequencing data
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#archival-of-data-code-and-results" title="Archival of data, code, and results" class="md-nav__link">
    Archival of data, code, and results
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#results" title="Results" class="md-nav__link">
    Results
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-12-cognitive-training-alters-spatial-approach-and-avoidance-behavior" title="Fig. 1.2: Cognitive training alters spatial approach and avoidance behavior." class="md-nav__link">
    Fig. 1.2: Cognitive training alters spatial approach and avoidance behavior.
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#conflict-trained-mice-exhibit-cognitive-discrimination" title="Conflict-trained mice exhibit cognitive discrimination" class="md-nav__link">
    Conflict-trained mice exhibit cognitive discrimination
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-13-subfield-differences-in-hippocampal-gene-expression" title="Fig. 1.3: Subfield differences in hippocampal gene expression" class="md-nav__link">
    Fig. 1.3: Subfield differences in hippocampal gene expression
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#confirmation-of-subfield-specific-gene-expression-patterns" title="Confirmation of subfield-specific gene expression patterns" class="md-nav__link">
    Confirmation of subfield-specific gene expression patterns
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#research-question-1-how-does-memory-associated-place-avoidance-alter-gene-expression-in-the-dorsal-hippocampus" title="Research question 1: How does memory-associated place avoidance alter gene expression in the dorsal hippocampus?" class="md-nav__link">
    Research question 1: How does memory-associated place avoidance alter gene expression in the dorsal hippocampus?
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-14-place-avoidance-is-associated-with-up-regulation-genes-involved-in-regulation-of-transcription" title="Fig. 1.4: Place avoidance is associated with up-regulation genes involved in regulation of transcription." class="md-nav__link">
    Fig. 1.4: Place avoidance is associated with up-regulation genes involved in regulation of transcription.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#research-question-2-does-cognitive-discrimination-alter-gene-expression" title="Research question 2: Does cognitive discrimination alter gene expression?" class="md-nav__link">
    Research question 2: Does cognitive discrimination alter gene expression?
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-15-place-avoidance-is-associated-with-increased-expression-of-genes-that-regulate-synaptic-activity-in-ca1" title="Fig. 1.5: Place avoidance is associated with increased expression of genes that regulate synaptic activity in CA1." class="md-nav__link">
    Fig. 1.5: Place avoidance is associated with increased expression of genes that regulate synaptic activity in CA1.
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#table-11-differentially-expressed-genes-p-01-by-cognitive-training-and-subfield" title="Table 1.1: Differentially expressed genes (p &lt; 0.1) by cognitive training and subfield." class="md-nav__link">
    Table 1.1: Differentially expressed genes (p &lt; 0.1) by cognitive training and subfield.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#research-question-3-does-unavoidable-punishment-in-the-form-of-random-mild-foot-shocks-alter-gene-expression" title="Research question 3: Does unavoidable punishment (in the form of random, mild foot-shocks) alter gene expression?" class="md-nav__link">
    Research question 3: Does unavoidable punishment (in the form of random, mild foot-shocks) alter gene expression?
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-16-additional-punishment-also-influences-gene-expression-in-ca1" title="Fig. 1.6: Additional punishment also influences gene expression in CA1." class="md-nav__link">
    Fig. 1.6: Additional punishment also influences gene expression in CA1.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#discussion" title="Discussion" class="md-nav__link">
    Discussion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#acknowledgments" title="Acknowledgments" class="md-nav__link">
    Acknowledgments
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
    
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../02_fmr1/" title="Chapter 2 Genetic manipulation" class="md-nav__link">
      Chapter 2 Genetic manipulation
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../03_dissociation/" title="Chapter 3 Neural Dissociation" class="md-nav__link">
      Chapter 3 Neural Dissociation
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../authors-notes.md" title="Author's notes" class="md-nav__link">
      Author's notes
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../conclusion/" title="Conclusion" class="md-nav__link">
      Conclusion
    </a>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../cv/" title="My CV" class="md-nav__link">
      My CV
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary">
  
  
  
    <label class="md-nav__title" for="__toc">Table of contents</label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#chapter-1-avoidance-learning-induces-strong-up-regulation-of-transcription-in-the-dg-subfield-of-the-dorsal-hippocampus" title="Chapter 1:  Avoidance learning induces strong up-regulation of transcription in the DG subfield of the dorsal hippocampus" class="md-nav__link">
    Chapter 1:  Avoidance learning induces strong up-regulation of transcription in the DG subfield of the dorsal hippocampus
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#abstract" title="Abstract" class="md-nav__link">
    Abstract
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#introduction" title="Introduction" class="md-nav__link">
    Introduction
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#materials-and-methods" title="Materials and Methods" class="md-nav__link">
    Materials and Methods
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#the-active-place-avoidance-task" title="The Active Place Avoidance Task" class="md-nav__link">
    The Active Place Avoidance Task
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#statistical-analyses-of-behavior" title="Statistical analyses of behavior" class="md-nav__link">
    Statistical analyses of behavior
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-11-experimental-design" title="Fig. 1.1: Experimental design." class="md-nav__link">
    Fig. 1.1: Experimental design.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tissue-preparation-from-dg-ca3-and-ca1-subfields" title="Tissue preparation from DG, CA3, and CA1 subfields" class="md-nav__link">
    Tissue preparation from DG, CA3, and CA1 subfields
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-sequencing-and-bioinformatics" title="RNA-sequencing and bioinformatics" class="md-nav__link">
    RNA-sequencing and bioinformatics
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#statistics-and-data-visualization-of-rna-sequencing-data" title="Statistics and data visualization of RNA-sequencing data" class="md-nav__link">
    Statistics and data visualization of RNA-sequencing data
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#archival-of-data-code-and-results" title="Archival of data, code, and results" class="md-nav__link">
    Archival of data, code, and results
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#results" title="Results" class="md-nav__link">
    Results
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-12-cognitive-training-alters-spatial-approach-and-avoidance-behavior" title="Fig. 1.2: Cognitive training alters spatial approach and avoidance behavior." class="md-nav__link">
    Fig. 1.2: Cognitive training alters spatial approach and avoidance behavior.
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#conflict-trained-mice-exhibit-cognitive-discrimination" title="Conflict-trained mice exhibit cognitive discrimination" class="md-nav__link">
    Conflict-trained mice exhibit cognitive discrimination
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-13-subfield-differences-in-hippocampal-gene-expression" title="Fig. 1.3: Subfield differences in hippocampal gene expression" class="md-nav__link">
    Fig. 1.3: Subfield differences in hippocampal gene expression
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#confirmation-of-subfield-specific-gene-expression-patterns" title="Confirmation of subfield-specific gene expression patterns" class="md-nav__link">
    Confirmation of subfield-specific gene expression patterns
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#research-question-1-how-does-memory-associated-place-avoidance-alter-gene-expression-in-the-dorsal-hippocampus" title="Research question 1: How does memory-associated place avoidance alter gene expression in the dorsal hippocampus?" class="md-nav__link">
    Research question 1: How does memory-associated place avoidance alter gene expression in the dorsal hippocampus?
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-14-place-avoidance-is-associated-with-up-regulation-genes-involved-in-regulation-of-transcription" title="Fig. 1.4: Place avoidance is associated with up-regulation genes involved in regulation of transcription." class="md-nav__link">
    Fig. 1.4: Place avoidance is associated with up-regulation genes involved in regulation of transcription.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#research-question-2-does-cognitive-discrimination-alter-gene-expression" title="Research question 2: Does cognitive discrimination alter gene expression?" class="md-nav__link">
    Research question 2: Does cognitive discrimination alter gene expression?
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-15-place-avoidance-is-associated-with-increased-expression-of-genes-that-regulate-synaptic-activity-in-ca1" title="Fig. 1.5: Place avoidance is associated with increased expression of genes that regulate synaptic activity in CA1." class="md-nav__link">
    Fig. 1.5: Place avoidance is associated with increased expression of genes that regulate synaptic activity in CA1.
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#table-11-differentially-expressed-genes-p-01-by-cognitive-training-and-subfield" title="Table 1.1: Differentially expressed genes (p &lt; 0.1) by cognitive training and subfield." class="md-nav__link">
    Table 1.1: Differentially expressed genes (p &lt; 0.1) by cognitive training and subfield.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#research-question-3-does-unavoidable-punishment-in-the-form-of-random-mild-foot-shocks-alter-gene-expression" title="Research question 3: Does unavoidable punishment (in the form of random, mild foot-shocks) alter gene expression?" class="md-nav__link">
    Research question 3: Does unavoidable punishment (in the form of random, mild foot-shocks) alter gene expression?
  </a>
  
    <nav class="md-nav">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#fig-16-additional-punishment-also-influences-gene-expression-in-ca1" title="Fig. 1.6: Additional punishment also influences gene expression in CA1." class="md-nav__link">
    Fig. 1.6: Additional punishment also influences gene expression in CA1.
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#discussion" title="Discussion" class="md-nav__link">
    Discussion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#acknowledgments" title="Acknowledgments" class="md-nav__link">
    Acknowledgments
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
      
      
      
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                
                  <h1>Chapter 1 Brains and behavior</h1>
                
                <h2 id="chapter-1-avoidance-learning-induces-strong-up-regulation-of-transcription-in-the-dg-subfield-of-the-dorsal-hippocampus">Chapter 1:  Avoidance learning induces strong up-regulation of transcription in the DG subfield of the dorsal hippocampus<a class="headerlink" href="#chapter-1-avoidance-learning-induces-strong-up-regulation-of-transcription-in-the-dg-subfield-of-the-dorsal-hippocampus" title="Permanent link">&para;</a></h2>
<p>See corresponding data and analysis here: <a href="https://github.com/raynamharris/IntegrativeProjectWT2015">https://github.com/raynamharris/IntegrativeProjectWT2015</a></p>
<h3 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h3>
<p>The hippocampus is widely known to be involved in spatial navigation, learning, and memory, but little is known about how dynamic activity of transcriptome supports these cognitive processes. I aimed to fill that gap by asking, “How do memory-associated place avoidance and cognitive discrimination alter gene expression in the dorsal hippocampus?”. This research required an interdisciplinary approach to utilize both the hypothesis-driven techniques of behavioral neuroscience and the discovery-driven methods of behavioral genomics to identify novel neuromolecular substrates of memory. I found that gene expression in CA1 and DG discriminates internal versus external variables because yoked and trained mice had the identical physical experience of the world but could interpret the experience differently. I observed that different gene functions and patterns of differential expression in the hippocampal subfields. DG most responsive to memory formation, and all the upregulated genes are known to be involved nuclear signaling; while changes in CA1 reflect membrane-level ion-channel regulation. The CA1 results also demonstrate sensitivity to the amount of unavoidable shock or stress that occurred 24h in the past, which could be a molecular signature for a form of stress-related memory. Finally, the results demonstrated that conflict learning does not cause additional gene expression changes relative to initial learning. The strength of this research is its collaborative, integrative, and reproducible approaches that provide a deeper understanding of the molecular changes that are or are not associated with robust behavioral output indicative of memory.</p>
<h3 id="introduction">Introduction<a class="headerlink" href="#introduction" title="Permanent link">&para;</a></h3>
<p>For centuries, scientists and philosophers have sought to understand how the brain uses memory to drive changes in behavior. One challenge is that memory cannot be physically isolated. Memory is an instance in which an organism's current behavior is determined by some aspect of its previous experience30. It an emergent property that researchers observe as an overt change at one or more levels of biological organization. Recent advances in molecular biology and neuroscience pushed our understanding past the concept of the neural doctrine (single neurons are the brain’s information processing unit of organization) and the central dogma of molecular biology (function is coded by the process DNA-&gt;RNA-&gt;Protein) which alone do not solve the problem of understanding how the brain stores and recalls memories that change animal behavior. </p>
<p>Neurons differ from one another in many ways: structurally, functionally and genetically, as well as the connections they make with other cells. The extent to which gene expression accounts for the function and characteristics of cell types, neural systems, and behavioral expression is unknown despite being fundamental to many research programs in biology and medicine. </p>
<p>One goal of modern neuroscience is to fully characterize the structure and function of all the molecules in all the neurons in all the circuits of the human brain (8,9 but see 7,31). Additionally, understanding how such variability and plasticity in the brain gives rise to emergent phenotypical variation is of utmost importance for advancing neuroscience and related fields. Generations of behavioral neuroscientists have developed sophisticated paradigms for assessing the biological correlates of memory and cognition14.  However, it can be a challenge to interpret the significance of observed behavior differences. “The problem with rats is that I can't ask them if they remember what they had for breakfast,” Howard Eichenbaum once said32. Nevertheless, community-developed standards were established such that robust behavioral phenotypes could be used as indicators of memory-associated changes in behavior. Memory is a term used to characterize instances in which an organism's current behavior is determined by some aspect of its previous experience30. In active avoidance paradigms, the focal animal is required to emit a specific response in order to escape or avoid a noxious stimulus33. Thus, avoidance behavior is a readable outcome of memory.</p>
<p>The hippocampus is widely known to be involved in spatial navigation, learning, and memory5. In 1909, Cajal illustrated the hippocampus with directional connections between neurons between  the entorhinal cortex (EC), subregions of Cornu Ammonius' Ammon's horn (CA) and the dentate gyrus (DG), with information flowing from in the EC -&gt; CA1 -&gt; CA3 -&gt; DG -&gt; EC or through additional pathways that skip subfields or layers within subfields34. In 1971, John O'Keefe found that place cells in the hippocampus function as an internal global positions system (GPS) in our brain5,6. Different place cells are active when an animal is in different locations within the environment35. In the 2000s, May-Britt Moser and Edvard Moser demonstrated the place cells of the CA1 were connected to grid cells of the entorhinal cortex11. The CA1 pyramidal cells are often referred to as place cells because they activate when an animal is in a particular place in its environment. The CA3-CA1 synapse has received a lot of attention for long-term potentiation (LTP), long-term depression (LTD), and other forms of synaptic plasticity as it might relate to learning. The DG has been given some attention for adult-born neurogenesis and its role in learning and memory36–39.</p>
<p>Since the early 80s with discoveries that the NMDA receptor and Calcium-dependent protein kinase II (CaMKII) play crucial roles in LTP, neuroscientists have been using pharmacology to identify the molecular and neural substrates of synaptic plasticity and memory. These studies linked changes in behavior to long-term potentiation, long-term depression, and the expression of numerous candidate genes40,41. Molecular and cellular biologists have shed light on what molecules serve as biomarkers for cells with different structure and function42–44. This tools have highlighted the heterogeneity of the structure and function of neurons and have lead to new hypothesis regarding how different neuron classes contribute to cognition. Modern genomic techniques such as microarrays and next-generation sequencing have made it possible to examine the molecular underpinnings of plasticity in animal behavior18,45–47. Yet despite its successes in generating discovery-driven insights, behavioral genomics has been criticized for its lack hypotheses48. Additionally, databases of gene expression for mouse hippocampi do exists49–53, but they provide little insight into how dynamic activity of transcriptome supports memory.</p>
<p>The goal of this research is to marry the hypothesis-driven techniques of behavioral neuroscience with the discovery-driven methods of behavioral genomics to identify novel neuromolecular substrates of memory. First, I analyzed the behavior of mice in a task explicitly designed to test how the brain uses spatial information for memory processing. My experiment used a variant of the active place avoidance task33,36,54–56 which allowed us to tease apart the effects of subtle differences in an animal's experience of the environment. Then, I characterized the transcriptional pattern of activity from tissue samples from the DG-CA3-CA1 tri-synaptic pathway that has been implicated in spatial learning and memory. The results of this work showed that this work confirms previously identified sub-field specific patterns of expression and confirms previous descriptions of candidate memory gene activity in CA1. Importantly, I also detected novel patterns of transcription factor activation in the DG and a surprising lack of transcriptional plasticity in CA3. These two findings suggest new mechanisms for how information flows through a DG-CA3-CA1 pathway that has been implicated in memory. The results provide a unique perspective on the role of transcriptional stability and plasticity about hippocampal-dependent learning and memory. In the spirit of reproducible research and open science57–60, all data, code, and results are publically available and licensed under a creative commons for reuse for the continued advancement of basic research.</p>
<h3 id="materials-and-methods">Materials and Methods<a class="headerlink" href="#materials-and-methods" title="Permanent link">&para;</a></h3>
<p>All animal care and use complies with the Public Health Service Policy on Humane Care and Use of Laboratory Animals and were approved by the New York University Animal Welfare Committee and the Marine Biological Laboratory Institutional Animal Care and Use Committee.  Male C57BL/6J mice were housed at the Marine Biological Laboratory on a 12:12 (light: dark) cycle with continuous access to food and water in home cages with up to five littermates. </p>
<h4 id="the-active-place-avoidance-task">The Active Place Avoidance Task<a class="headerlink" href="#the-active-place-avoidance-task" title="Permanent link">&para;</a></h4>
<p>To examine spatial learning and memory, we used a well-established active place avoidance paradigm36,54,61. Littermates were randomly assigned to one of our treatment groups (nyoked-consistent=8, nconsistent=8, nyoked-conflict=9, nconflict=9). All mice were exposed to nine sessions in the active place avoidance arena (Fig. 1.1A). Mice were placed on an elevated circular 40-cm diameter arena that rotated at 1 rpm. The arena wall was transparent and thus contained the mice to the arena while allowing it to observe the environment. The location of the mouse in the arena was determined from an overhead television camera a PC-controlled tracking system (Bio-Signal Group). Consistently trained mice in the active place avoidance task are conditioned to avoid mild shocks (constant current 0.2 mA 500 ms 60 Hz) that can be localized by visual cues in the environment. Yoked mice are delivered a sequence of unavoidable shocks that reproduces the time series of shocks received by the trained mice; however, the shocks delivered to the mice cannot be localized by visual cues in the environment. All sessions in the arena last 10 minutes. Mice are allowed to become familiar with walking on a rotating arena during a 10 min pretraining session. Then mice undergo 3, 10-min avoidance training sessions separated by a 2 h inter-trial interval.  Mice are returned to their home cage overnight. The next day, mice are subjected to a 10-min ‘Retest session’ where the shock is in the same location as before. For the next three training sessions, the shock zone remains in the same place for consistently trained animals, but it is rotated 180° for the conflict-trained mice. The next day, all mice are subjected to a 1-min “Retention session” with the shock off. </p>
<h4 id="statistical-analyses-of-behavior">Statistical analyses of behavior<a class="headerlink" href="#statistical-analyses-of-behavior" title="Permanent link">&para;</a></h4>
<p>Place avoidance was evaluated by end-point measures output by TrackAnalysis software. Forty quantitative variables were measured that capture the animals’ use of space and time. The two estimates of place avoidance used were the reduction in entrances into the shock zone and the increased time to stay out of the shock zone. A repeated measures ANOVA with sphericity correction was used to identify group differences in behavioral measure across all training sessions62,63. A two-way ANOVA was used to compare the mean difference between conflict and consistently trained mice during a single training session62. The non-parametric Kendall's tau statistic was used to estimate a rank-based measure of correlation between the number of entrances and maximum avoidance time62.  </p>
<p><img alt="fig1" src="https://github.com/raynamharris/Thesis/blob/master/docs/_images/11_method.png?raw=true" /></p>
<h5 id="fig-11-experimental-design">Fig. 1.1: Experimental design.<a class="headerlink" href="#fig-11-experimental-design" title="Permanent link">&para;</a></h5>
<p><em>A) Mice were assigned to one of four groups: consistently-trained (red, n=8), yoked-consistent (dark grey, n=8), conflict-trained (peach, n=9), or yoked-conflict (light grey, n=9). Mice were placed on the rotating arena (1 rpm) for training sessions that lasted 10 min and was separated by 2-hour intersession interval or overnight (~17 hr). Behavior was recorded during the Pre-training, Training (T1-T6), Retest, and Retention session. In the active place avoidance schematics, the shaded pie-shaped region is the behaviorally relevant region used for counting the number of entrances into the shock zone. The shocking of yoked mice is not spatially limited to the dark-grey pie-shaped zone, but consistent and conflict trained mice only receive shocks in the red and peach-shaded regions, respectively. B) A representative photo shows the size and location of tissue samples collected for RNA-sequencing.  C) Graphical illustration of hippocampal tissues sequenced and sample sizes for each treatment group and hippocampal subfield.</em></p>
<h4 id="tissue-preparation-from-dg-ca3-and-ca1-subfields">Tissue preparation from DG, CA3, and CA1 subfields<a class="headerlink" href="#tissue-preparation-from-dg-ca3-and-ca1-subfields" title="Permanent link">&para;</a></h4>
<p>Thirty minutes after the last cognitive training session, mice were anesthetized with 2% (vol/vol) isoflurane for 2 minutes and decapitated. Transverse 300 μm brain slices were cut using a vibratome (model VT1000 S, Leica Biosystems, Buffalo Grove, IL) and incubated at 36°C for 30 min and then at room temperature for 90 min in oxygenated artificial cerebrospinal fluid (aCSF in mM: 125 NaCl, 2.5 KCl, 1 MgSO4, 2 CaCl2, 25 NaHCO3, 1.25 NaH2PO4 and 25 Glucose) 64,65. The DG, CA3, CA1 subfields were microdissected using a 0.25 mm punch (Electron Microscopy Systems) and a Zeiss dissecting scope (Fig. 1B). RNA was isolated using the Maxwell 16 LEV RNA Isolation Kit (Promega). RNA libraries were prepared by the Genomic Sequencing and Analysis Facility at the University of Texas at Austin and sequenced on the Illumina HiSeq platform.</p>
<h4 id="rna-sequencing-and-bioinformatics">RNA-sequencing and bioinformatics<a class="headerlink" href="#rna-sequencing-and-bioinformatics" title="Permanent link">&para;</a></h4>
<p>Raw reads were from the transferred from the GSAF to the Stampede Cluster at the Texas Advanced Computing Facility (TACC) via Amazon Cloud. Quality of the data was checked using the program FASTQC66 and visualized using MultiQC67. Samples sizes for each treatment group and subfield are reported in Figure 1C. I obtained a median number of 7.3 million reads per samples, with a maximum of 37 million and a minimum of 1.5 million reads. Next, I used the program Kallisto68 for pseudo-alignment of raw reads to a mouse references transcriptome (Gencode version 7)69. A median number of 2.25 million reads were pseudo-aligned to the references transcriptome. To confirm that Kallisto performed well on raw reads, I also removed low-quality reads and contaminating adapter sequences using the program Cutadapt70, but these trimmed and filtered reads yield less than 1 million mapped reads for all but 3 of the 44 samples; therefore all subsequent analyses were conducted on the pseudo-aligned raw reads.</p>
<h4 id="statistics-and-data-visualization-of-rna-sequencing-data">Statistics and data visualization of RNA-sequencing data<a class="headerlink" href="#statistics-and-data-visualization-of-rna-sequencing-data" title="Permanent link">&para;</a></h4>
<p>Transcript counts from Kallisto were imported into R62 and aggregated to yield gene counts using the ‘gene’ identifier from the Gencode transcriptome. I used DESeq271 to normalize and quantify gene counts with a false discovery corrected (FDR) p-value &lt; 0.1. The DESeq2 models including subfield (DG, CA3, CA1), training group (yoked-consistent, consistent, yoked-conflict, and conflict) and their interaction. Hierarchical clustering by correlation and volcano plots were used to visualize the patterns of differential gene expression71–76. Principal component analysis (PCA) was conducted to reduce the dimensionality of the data, and ANOVAs were used to test for group differences in the principal components. A chi-squared goodness of fit test was used to test for equal distribution of up- and down-regulated gene expression between two-way contrasts62.  I used GO_MWU77 to identify gene ontology categories using a -log(p-value) as a continuous measure of significance that are significantly enriched with either up- or down-regulated genes for a given two-way contrast. I used ggplot276, cowplot75, pheatmap78, viridis79, ColorBrewer72, and colorblindr80 to make figures that are (hopefully) color-blind friendly. Multi-panel figures and illustrations were created using Adobe Illustrator.</p>
<h4 id="archival-of-data-code-and-results">Archival of data, code, and results<a class="headerlink" href="#archival-of-data-code-and-results" title="Permanent link">&para;</a></h4>
<p>I archived the raw sequence data and intermediate data files in NCBI's Gene Expression Omnibus Database (accession: GSE99765). The data and code are publically available on GitHub  with a stable version archived at Zenodo 81. </p>
<h3 id="results">Results<a class="headerlink" href="#results" title="Permanent link">&para;</a></h3>
<p>Conflict- and consistently-trained mice exhibit place avoidance
Active place avoidance is evidenced by a reduction in the mean number of entrances into the shock zone (F(24,240) = 5.140, p = 5.57e-08) (Fig. 1.2A). Place avoidance is also evidenced by an increase in the time the mouse stays out of the conditioned shock zone, as illustrated by time to the second entrance (F(24,240) = 5.30, p = 2.20e-09) (Fig. 1.2B). These two measures of behavior (number of entrances and time to second entrance) are inversely correlated (tau=-0.55, p &lt; 2.2e-16). A principal component analysis of all the quantitative variables shows that the two measures load strongly onto PC1, which is significantly different between treatments groups (F3=70.92, p = 1.01e-13) (Fig. 1.2C).  In fact, most of the quantitative measures captured by the video tracking software are positively or negatively correlated with the measures of shock zone entrances or avoidance, as evidenced by a hierarchical clustering analysis (Fig. 1.2C). Notably, measures of speed are not correlated with place avoidance, but there is a pattern that speed is highest during the pre-training session compared to the later sessions.   </p>
<p><img alt="fig2" src="https://github.com/raynamharris/Thesis/blob/master/docs/_images/12_behavior.png?raw=true" /></p>
<h5 id="fig-12-cognitive-training-alters-spatial-approach-and-avoidance-behavior">Fig. 1.2: Cognitive training alters spatial approach and avoidance behavior.<a class="headerlink" href="#fig-12-cognitive-training-alters-spatial-approach-and-avoidance-behavior" title="Permanent link">&para;</a></h5>
<p><em>A) Consistently trained (red lines) mice make fewer entrances into the shock zone than yoked-mice (dark grey lines) on all training (T1-T6), restest, and retention (Reten.) sessions but not during the pre-training session (Pre.). Conflict-trained mice (peach) and their yoked controls (light grey) show a similar pattern except for that mean number of differences between T1 and T4 do not differ between conflict-trained mice. B) Time to second shock zone entrance shows a pattern that is reciprocal to the mean number of entrances. C) A principal component analysis estimates that cognitive training explains 36% of the observed variation in behavior (red and peach versus dark grey and light grey). Among the top five contributing variables are the number of entrances and the max avoidance time. D) Hierarchical clustering by correlation of 40 behaviors shows that approach, latency to approach (or avoidance), and speed are primary behavioral variables captured by our video-tracking software. Clustering distinguishes trained and yoked animals but does not provide precise temporal resolution. The color scale shows centered z-s for high (yellow) and low (deep purple) values for each quantitative variable. For code see <a href="https://github.com/raynamharris/IntegrativeProjectWT2015/blob/master/scripts/01a_beahvior_analysis.md">Github</a>.</em></p>
<h4 id="conflict-trained-mice-exhibit-cognitive-discrimination">Conflict-trained mice exhibit cognitive discrimination<a class="headerlink" href="#conflict-trained-mice-exhibit-cognitive-discrimination" title="Permanent link">&para;</a></h4>
<p>Cognitive discrimination requires distinguishing between similar but distinct experiences. I investigated cognitive discrimination by changing the location of shock. Conflict-trained and consistently-trained mice differ in the mean number of entrances during the T4/C1 session (F(1) =17.49, p=0.000801), but not during the T6/C3 session (F(1) = 0.265, p=0.614) (Fig. 1.2A). These results indicate that, with continued training, the conflict mice rapidly learned to discriminate between the memories of the old and new shock locations.</p>
<p><img alt="fig3" src="https://github.com/raynamharris/Thesis/blob/master/docs/_images/13_subfield.png?raw=true" /> </p>
<h5 id="fig-13-subfield-differences-in-hippocampal-gene-expression">Fig. 1.3: Subfield differences in hippocampal gene expression<a class="headerlink" href="#fig-13-subfield-differences-in-hippocampal-gene-expression" title="Permanent link">&para;</a></h5>
<p><em>A) I compared gene expression in three hippocampal subfields from our four treatment groups (DG: orange, CA3: green, CA1: purple, yoked-consistent: filled circle, consistent: open square, yoked-conflict: filled square, conflict: open square). B) Hierarchical clustering of differentially expressed genes shows variation between subfields is much greater than variation induced by treatment. C) A principal component analysis estimates that over 50 % of the variation is capture in PC1 and P3, which visually separate the three hippocampal subfields. D) 3000 are differentially expressed in a symmetric pattern between DG and CA. E, F) Fewer genes are up-regulated in CA1 compared to both DG and CA3, but the magnitude of expression differences in greater between DG-CA1 than between CA3-CA1. For volcano plots, dots are partially transparent to aid visualization of density. For code see <a href="https://github.com/raynamharris/IntegrativeProjectWT2015/blob/master/scripts/02b_rnaseqALL.md">Github</a>.</em></p>
<h4 id="confirmation-of-subfield-specific-gene-expression-patterns">Confirmation of subfield-specific gene expression patterns<a class="headerlink" href="#confirmation-of-subfield-specific-gene-expression-patterns" title="Permanent link">&para;</a></h4>
<p>Large differences in subfield specific gene expression are well documented49–53, but the association of memory and gene expression is understudied. Thus, I examined broad patterns of gene expression variation in DG, CA3, and CA1 tissue samples from mice in each of the four cognitive training groups (Fig. 1.1C, Table 1.2A). My results confirm previous studies by showing significant differences in gene expression of thousands of genes between brain regions (Fig. 1.3). Hierarchical clustering of the top 250 differentially expressed genes at FDR 0.1 reveals a strong signature of subfield-specific expression, with all samples clustering by subfield (Fig. 1.3B). PC1 and PC2 account for 71% of the variation and are both significantly different between subfields (PC1 - F2:41=256.2, p &lt;&lt; 0.001; PC2 - F2:41=1030, p &lt;&lt; 0.001). PC6 explains only 1% of the variation in gene expression it does vary according to cognitive training (F3:40=12.01, p&lt;&lt;0.001). </p>
<p>Hierarchical clustering and principal component analyses are a convenient way to visualize gene expression differences between all samples, but volcano plots are a convenient way to explore two-way contrasts in more detail. The contrast between brain regions highlights the magnitude of differential expression between subfields (DG vs. CA3: 3145, DG vs. CA1: 2526, CA3 vs. CA1: 2022 differentially expressed genes at FDR=0.1). The distribution differential gene expression is symmetrical between CA3 and DG (Fig 1.3D). Fewer genes that half the differentially expressed genes are up-regulated in CA1 compared to DG (Fig 1.3E) and CA3 (Fig 1.3F), but the magnitude of expression differences in greater between DG-CA1 (Fig 1.3F).</p>
<h4 id="research-question-1-how-does-memory-associated-place-avoidance-alter-gene-expression-in-the-dorsal-hippocampus">Research question 1: How does memory-associated place avoidance alter gene expression in the dorsal hippocampus?<a class="headerlink" href="#research-question-1-how-does-memory-associated-place-avoidance-alter-gene-expression-in-the-dorsal-hippocampus" title="Permanent link">&para;</a></h4>
<p>All analyses of the effect of training on hippocampal gene expression were conducted independently for each subfield. In the DG, 116 genes were upregulated in the consistently training group while 0 were upregulated in the yoked group (Fig 1.4A, Table 1.2). Among the top ten differentially expressed genes are those encoding transcription factors Egr4, Junb, SMAD. A gene ontology analysis shows an enrichment in molecular functions related to core promoter binding, nuclear localization sequence binding, signal sequence binding, poly(A) RNA binding, and heat-shock protein binding (Fig 1.4B). No genes in CA3 were differentially expressed in the consistently trained vs. yoked contrast (Table 1.2). This lack of activity is consistent with the Denny et al. study who expressed Cre-ERT2 under the direction of the activity regulated cytoskeletal-associated protein (Arc) promoter regions to compare activation of neuronal populations in the hippocampus during encoding and retrieval of memory39. However, in CA1, approximately 600 genes were differentially expressed between trained vs. yoked contrasts (Fig. 1.5A, Table 1.2). The distribution of up and down-regulation of genes is symmetric in the CA1 subfield. An analysis of gene ontology categories shows an enrichment for molecular functions related to ion channel synthesis and activity (Fig. 1.5B).   </p>
<p><img alt="fig4" src="https://github.com/raynamharris/Thesis/blob/master/docs/_images/14_DG.png?raw=true" /></p>
<h5 id="fig-14-place-avoidance-is-associated-with-up-regulation-genes-involved-in-regulation-of-transcription">Fig. 1.4: Place avoidance is associated with up-regulation genes involved in regulation of transcription.<a class="headerlink" href="#fig-14-place-avoidance-is-associated-with-up-regulation-genes-involved-in-regulation-of-transcription" title="Permanent link">&para;</a></h5>
<p><em>A) In the dentate gyrus (DG) 116 genes are upregulated in the consistently training group compared to the yoked samples (FDR = 0.1). An analysis of enrichment in gene ontology categories shows an enrichment in molecular function processes related to promoter binding and nuclear sequence binding (p &lt; 0.05).  B) Genes and GO categories are colored according to enrichment in trained (red) or yoked (black). The active place avoidance schematics, the shaded pie-shaped region is the behaviorally relevant region for counting a number of entrances into the shock zone. Trained (red) mice are shocked in this zone, but the shocking of yoked mice is not spatially limited to the dark-grey pie-shaped zone. For code see <a href="https://github.com/raynamharris/IntegrativeProjectWT2015/blob/master/scripts/02c_rnaseqSubfield.md">Github</a>.</em></p>
<h4 id="research-question-2-does-cognitive-discrimination-alter-gene-expression">Research question 2: Does cognitive discrimination alter gene expression?<a class="headerlink" href="#research-question-2-does-cognitive-discrimination-alter-gene-expression" title="Permanent link">&para;</a></h4>
<p>Changing the shock zone location provides a test of cognitive discrimination that requires DG function36,82. Once an animal learns the new location of shock, cognitive discrimination enables judicious use knowledge of the current and former locations of shock. Yoked, consistently-trained, and conflict-trained animals vary in their degree of behaviors expressed that evidence cognitive discrimination. We asked if changes in gene expression are associated with cognitive discrimination. We found gene expression in the DG, CA3, and CA1 shows a remarkable lack of differential expression in response conflict training (Table 1.1). The results show that initial learning of the shock zone does initial learning of the shock zone does change synaptic function in the perforant path DG and in the pyramidal layer of CA1; however, less than 1/1000 of the transcriptome is differentially expressed in the animals that behaviorally demonstrated cognitive discrimination in relation. </p>
<p><img alt="fig5" src="https://github.com/raynamharris/Thesis/blob/master/docs/_images/15_CA1.png?raw=true" /> </p>
<h5 id="fig-15-place-avoidance-is-associated-with-increased-expression-of-genes-that-regulate-synaptic-activity-in-ca1">Fig. 1.5: Place avoidance is associated with increased expression of genes that regulate synaptic activity in CA1.<a class="headerlink" href="#fig-15-place-avoidance-is-associated-with-increased-expression-of-genes-that-regulate-synaptic-activity-in-ca1" title="Permanent link">&para;</a></h5>
<p><em>A) In the CA1, 253 genes are upregulated in the consistently training group while only 255 are downregulated (FDR = 0.1). B) An analysis of enrichment in gene ontology (GO) categories shows an enrichment in molecular function processes related to ion channel transport and activity (p &lt; 0.05). Genes and GO categories are colored according to enrichment in trained (red) or yoked (black). For code see <a href="https://github.com/raynamharris/IntegrativeProjectWT2015/blob/master/scripts/02e_GO_MWU/GO_MWU.md">Github</a>.</em></p>
<table>
<thead>
<tr>
<th>Two-way contrasts between groups</th>
<th align="center">DG</th>
<th align="center">CA3</th>
<th align="center">CA1</th>
</tr>
</thead>
<tbody>
<tr>
<td>consistent vs. yoked-consisten</td>
<td align="center">116</td>
<td align="center">0</td>
<td align="center">508</td>
</tr>
<tr>
<td>conflict vs. consistent</td>
<td align="center">0</td>
<td align="center">2</td>
<td align="center">0</td>
</tr>
<tr>
<td>yoked-conflict vs. yoked-consistent</td>
<td align="center">1</td>
<td align="center">1</td>
<td align="center">409</td>
</tr>
<tr>
<td>conflict vs. yoked-conflict</td>
<td align="center">4</td>
<td align="center">0</td>
<td align="center">0</td>
</tr>
</tbody>
</table>
<h5 id="table-11-differentially-expressed-genes-p-01-by-cognitive-training-and-subfield">Table 1.1: Differentially expressed genes (p &lt; 0.1) by cognitive training and subfield.<a class="headerlink" href="#table-11-differentially-expressed-genes-p-01-by-cognitive-training-and-subfield" title="Permanent link">&para;</a></h5>
<p><em>Consistent training alters the expression of ~100 and ~200 genes in DG and CA1, respectively. Conflict training has almost no effect on hippocampal subfield expression relative to its yoked counterpart nor to the consistently trained animals.</em></p>
<h4 id="research-question-3-does-unavoidable-punishment-in-the-form-of-random-mild-foot-shocks-alter-gene-expression">Research question 3: Does unavoidable punishment (in the form of random, mild foot-shocks) alter gene expression?<a class="headerlink" href="#research-question-3-does-unavoidable-punishment-in-the-form-of-random-mild-foot-shocks-alter-gene-expression" title="Permanent link">&para;</a></h4>
<p>The yoked-conflict group received more foot-shocks on day two when the conflict animals were performing the cognitive discrimination task.  The CA1 but not the DG or CA3 shows gene expression response to differing levels of punishment (Table 1.1). In the CA1, I identified differentially expressed of 409 genes between yoked groups that received different amounts of punishment (FDR = 0.1) (Fig. 1.6A). The volcano plots show a near symmetric distribution of genes that are higher in the consistent yoked (left side, dark-grey) and conflict yoked (right side, light grey). Analysis of enrichment in gene ontology (GO) categories shows an enrichment in molecular function processes related to glutamate receptors, signal transduction, ion channel transport in the conflict-yoked group; whereas the consistent yoked group showed an enrichment in processes related to RNA binding and ribosomal activity (p &lt; 0.05) (Fig. 1.6B).</p>
<p><img alt="fig6" src="https://github.com/raynamharris/Thesis/blob/master/docs/_images/16_CA1yoked.png?raw=true" /> </p>
<h5 id="fig-16-additional-punishment-also-influences-gene-expression-in-ca1">Fig. 1.6: Additional punishment also influences gene expression in CA1.<a class="headerlink" href="#fig-16-additional-punishment-also-influences-gene-expression-in-ca1" title="Permanent link">&para;</a></h5>
<p><em>A) In the CA1, 409 genes are differentially expressed in between yoked groups that received different amounts of punishment.  (FDR = 0.1). B) An analysis of enrichment in gene ontology (GO) categories shows an enrichment in molecular function processes related to ion channel transport and synaptic activity (p &lt; 0.05). Genes and GO categories are colored according to enrichment in yoked-conflict (light grey) or yoked-consistent (dark-grey). For code  see <a href="https://github.com/raynamharris/IntegrativeProjectWT2015/blob/master/scripts/02e_GO_MWU/GO_MWU.md">Github</a>.</em> </p>
<h3 id="discussion">Discussion<a class="headerlink" href="#discussion" title="Permanent link">&para;</a></h3>
<p>My approach combines hypothesis-driven and data-driven techniques to yield new insight into the dynamic nature of the hippocampus. I observed behaviors subservient of learning, memory, and cognitive discrimination in our experimental groups of mice, and we identified associated changes in the expression of specific molecular pathways in subfields hippocampus using RNA sequencing. I filled the gap by being the first to conduct an unbiased RNA sequencing screen of genes related to subfields of the hippocampus. The brain-wide scans of the Allen Institute and Cembrowski et al. were unbiased in their molecular discovery, as they examined behaviorally naïve mice and so did not incorporate rigorous behavioral assays. The exquisitely detailed work of a cadre of hippocampal neuroscientists described single genes or pathways in detail, but they did not measure the activity of the entire transcriptome. </p>
<p>This approach led to the following finding and interpretations.  Gene expression in was DG most responsive to memory formation, which supports previous findings of physiological plasticity in the region Park et. 2015. One can conclude that gene expression discriminates internal versus external variables because yoked and trained mice had the identical physical experience of the world but could interpret the experience differently – gene expression in CA1 and DG sensitive to this. Different types of genes are differentially regulated in DG and CA1. Change in DG is related to nuclear signaling and expression of immediately early genes while changes in CA1 reflect membrane-level ion-channel regulation. The CA1 results also demonstrate sensitivity to the amount of unavoidable shock or stress that occurred 24h in the past. Importantly, tissue was sampled at a time when cortisol levels are very likely to be equal83,84. This CA1 response could be a molecular signature for a form of stress ‘memory’ (i..e. a persistent (24-h) change in the system as a function of experience even if the mouse does not express it by long-term behavior or physiology). Additionally, I did detect changes in protein kinases, but I did not detect any differences in the variant PKMz or PKC, even thought Pastalkova et al. 2006 and others have shown that DG synaptic plasticity and place avoidance memory is crucially mediated by PKMz85. Finally, the results demonstrated that conflict learning does not cause additional gene expression changes relative to initial learning. Perhaps the changes associated with memory formation and behavioral change occur within a network of cells that were not captured in our tissue samples.</p>
<p>This research supports the use of transcription factors (often referred to as immediate early genes) such as Jun, Arc, c-Fos, have previously been shown to be significantly increased in hippocampal neurons in a novel environment or context86–88. Arc has been used to identify subpopulations of neurons that were activated during acquisition or retrieval of memory under a number of conditions39.</p>
<p>There are several limitations to our experimental design. First of all, gene expression was only measured at one time-point. Thus I cannot say what the activity looks like during different stages of memory acquisition and recall. Secondly, I measured activity within many neurons and other cell types in a tissue sample of the hippocampal subfields. This research provided more insight into the subfield specificity of the hippocampus, but single-cell techniques will be needed to tease apart differences among cell types or cells with differencing activity states. Thirdly, I did not conduct an unbiased screen of all subfields of the hippocampus. This study overlooked the CA2, CA4, and neighboring cortical regions that send and receive information from the hippocampus. Finally, when looking for small effects, it is essential to have a sufficient sample size, so we opted to sequence more biological replicates from a few brain regions than the other way around. In the end, a small sample size does limit the power of our experiment to identify gene expression changes associated with memory. A challenge for the future will be to capture and understand molecular activity in the hippocampus before, during, and after learning.   </p>
<p>The strength of this research is its collaborative and integrative approach. We brought hypotheses and expertise from relating disciplines of biology to address current problems in neuroscience and genomics. Thus, we now have a deeper understanding of the molecular changes that are or are not associated with robust behavioral output indicative of memory. I also applied best practices in open and reproducible research to provide a robust pipeline that can be re-used. The raw and processed data, analysis pipelines, results, and interpretations are available for download, enabling the curious to export data into their environment for more specialized analyses. This dataset expands upon and complement other existing publicly available gene expression databases, mostly notably Lein et al., 2007 and Cembrowski et al., 2016.</p>
<h3 id="acknowledgments">Acknowledgments<a class="headerlink" href="#acknowledgments" title="Permanent link">&para;</a></h3>
<p>I thank Laura Colgin, Mariana Rodriguez, and Eric Brenner for comments on earlier versions of this manuscript. I thank Becca Young Brim, Caitlin Freisen, Tessa Solomon Lane, Mariana Rodriguez, Eric Brenner, and Issac Miller-Crews for helping advice regarding on figures and oral presentations of this research. Thanks to the general R scienticific community for openly sharing your software, for making useful tutorials, and for being responsive to user-requests for new functionality.</p>
                
                  
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid">
        
          <a href="../00_intro/" title="Introduction" class="md-flex md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-back md-footer-nav__button"></i>
            </div>
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                Introduction
              </span>
            </div>
          </a>
        
        
          <a href="../02_fmr1/" title="Chapter 2 Genetic manipulation" class="md-flex md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-flex__cell md-flex__cell--stretch md-footer-nav__title">
              <span class="md-flex__ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                Chapter 2 Genetic manipulation
              </span>
            </div>
            <div class="md-flex__cell md-flex__cell--shrink">
              <i class="md-icon md-icon--arrow-forward md-footer-nav__button"></i>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
          <div class="md-footer-copyright__highlight">
            Copyright &copy; 2017 <a href="https://raynamharris.github.io/">Rayna Harris
          </div>
        
        powered by
        <a href="https://www.mkdocs.org">MkDocs</a>
        and
        <a href="https://squidfunk.github.io/mkdocs-material/">
          Material for MkDocs</a>
      </div>
      
        
  <div class="md-footer-social">
    <link rel="stylesheet" href="../assets/fonts/font-awesome.css">
    
      <a href="https://github.com/raynamharris" class="md-footer-social__link fa fa-github-alt"></a>
    
      <a href="https://twitter.com/raynamharris" class="md-footer-social__link fa fa-twitter"></a>
    
      <a href="https://linkedin.com/in/rayna-harris-484802ba" class="md-footer-social__link fa fa-linkedin"></a>
    
  </div>

      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/application.e72fd936.js"></script>
      
      <script>app.initialize({version:"1.0",url:{base:".."}})</script>
      
    
    
      
    
  </body>
</html>