## Introduction

### Problem Statement

How do we learn our way from one place to another? How do we remember places we have visited before? How do we remember our experiences in those places? These questions of learning, memory, and behavior apply to our everyday lives and can be studied rigorously with cellular and molecular approaches. Learning is influenced by many different biological timescales - physiological time via effects on brain activity through neural circuits, developmental via changes that effect genome modification and development, and evolutionary, via the processes that shape natural selection.

Neurons differ from one another in many ways: structurally, functionally and genetically, as well as the connections they make with other cells. Diversity in neuronal structure and function is a striking sources of variability in organisms with nervous systems and gives rise to complex behavior and cognition4. The mammalian hippocampus is a crucial brain system for processing and storing neural representations of complex information that includes memory, context, space, and time5,6. The extent to which gene expression accounts for the functional and characteristics of cell types, neural systems, and behavioral expression is unknown despite being fundamental to many research programs in biology and medicine. Understanding how such cellular variability in the brain gives rise to phenotypical variation is of utmost importance for advancing neuroscience and related fields7, yet we remain naïve about what is and is not stable. 

One goal of modern neuroscience is to fully characterize the structure and function of all the molecules in all the neurons in all the circuits of the human brain8,9, and a goal of modern biologists is sequence all the genes in all the tissues of millions of organisms.  Generations of behavioral neuroscientists have developed highly-sophisticated paradigms for taking a reductionists approach to neuroscience5,6,10–14.

### Neurogenomic approach to understanding brain function

On the other hand, the fields of data science and neurogenomics explore complex interactions among genotypes, phenotypes, and the environment by conducting large-scale studies with multiple hypotheses testing and data-driven discovery of unknown biological processes. Neurogenomics is a collaborative, interdisciplinary, integrative field of study that unites diverse expertise, techniques, theories, and approaches to shed new lights on complex biological processes. Integrating data from data-driven collection platforms like high-throughput sequencing and real-time behavioral tracking provides additional insight into how neural systems work. Conducting reproducible research in an open science environment can make this approach even more powerful.  

Large-scale transcriptomic studies have uncovered patterns of gene networks that reflect the biological phenotype15,16. Therefore, one of the advantages of using genome-scale transcriptomic tools, such as RNA-seq to study variability and plasticity is that the activity of the entire genome can be studied at once, and the genes can be classified according to their responses to specific manipulations and environmental variables. Genes that respond to environmental or experimental cue can be identified, and their biological and genomic characteristics studied, resulting in the analysis of higher-order biological processes linked to phenotypic variability17–19. Our understanding of the molecular processes underlying neuronal individuality and plasticity is still insufficient and made all the more difficult because we now know that experience changes gene expression, which itself changes neural function and thus the subsequent neural functions in future experience20–22.

Transcriptomic analyses examining sub-fields of the hippocampus (or the whole hippocampus) have shed light on proximate mechanisms regulating learning and memory23–28. However, the hippocampus and the CA1 subfield are composed of multiple morphologically and functionally distinct cell classes, which may make whole hippocampus samples poor estimates of the molecular activity patterns in learning-recruited neurons. How representative are hippocampal patterns of gene expression compared to single-neuron patterns of molecular activity? While imaging and molecular genetic tools have long complemented single cell electrophysiological analyses, it has now also become possible to conduct molecular level analyses on a single neuron level20. RNA-sequencing technology has advanced sufficiently in recent years to conduct single-cell transcriptomic projects. Indeed, several proof-of-principle studies demonstrating feasibility and robustness have been published. We will take both single neuron and hippocampal CA1 tissue samples to identify the extent to which patterns of variability are preserved across these two levels of analysis.

### Research approach

For my thesis, I use an integrative approach to fill a gap in our understanding of how the outward expression of learning and memory is controlled by cellular mechanisms in the brain (Fig 0.2). In Chapter 1, I will research how avoidance behaviors are regulated by molecular changes that alter the synaptic activity in a hippocampal circuit. My null hypothesis is that there is no difference in subfield specific response of hippocampal transcriptomes to conditioned place avoidance memory. The detection of subfield-specific changes in gene expression while shedding new light on our understanding of functional changes subservient to memory underlying changes in behavior.

In Chapter 2, I will examine the effects of genetic manipulation the fragile x mental retardation gene (Fmr1) on avoidance behavior, synaptic physiology in a hippocampal circuit, and gene expression in hippocampal subfields. Fmr1recieves a lot of attention from the autism research community, but much of the work has focused on translational or downstream effects, given FMRPs know role in regulating synaptic translation. The present research will provide a systems view of genetic manipulation that may provide insight for improved precision of tool for molecular and genetic manipulation of organisms. 

In Chapter 3, I will examine transcriptional activity in hippocampal subfields in response behavioral and technical manipulations. A well-designed behavioral paradigm allows me to explore the effect subtlety different learning experiences on neuromolecular activity. I aim to distinguish technical and biological variation in hippocampal gene expression profiling studies. This topic will be a significant contribution to our understanding of how experimental techniques may or may not mask our ability to identify biologically meaningful signatures of gene expression.

Finally, the appendix summarizes projects that reflect the scholarly environment in which this research was conducted. In the concluding chapter, I synthesize what I have learned from this body of research, what it means for different audiences and my vision for the future.

![Graphical abstract](https://github.com/raynamharris/Thesis/blob/master/docs/_images/01_abstract.png?raw=true)
